src/grpc/errors.ts

```diff
@@ -50,7 +50,7 @@ class ErrorGRPCServerVerification<T> extends ErrorGRPC<T> {
   exitCode = sysexits.UNAVAILABLE;
 }
 
-class ErrorPolykeyRemote<T> extends ErrorPolykey<T> {
+class ErrorPolykeyRemoteOLD<T> extends ErrorPolykey<T> {
```

Comment:
Why does this still exist? Is it for the agent RPC?

Reply:
It's the old GRPC stuff, the `agent` RPC still needs it to function. Stuff like this will be fully removed at the end of the agent migration stage.